core/math/math_funcs.h

```diff
@@ -187,6 +187,19 @@ class Math {
 #endif
 	}
 
+	static _ALWAYS_INLINE_ int32_t division_round_up(int32_t num, int32_t den) {
+		return (num + den - 1) / den;
+	}
+	static _ALWAYS_INLINE_ uint32_t division_round_up(uint32_t num, uint32_t den) {
+		return (num + den - 1) / den;
+	}
+	static _ALWAYS_INLINE_ int64_t division_round_up(int32_t num, int64_t den) {
+		return (num + den - 1) / den;
+	}
+	static _ALWAYS_INLINE_ uint64_t division_round_up(uint32_t num, uint64_t den) {
+		return (num + den - 1) / den;
```

Comment:
```suggestion
	static _ALWAYS_INLINE_ int32_t division_round_up(int32_t p_num, int32_t p_den) {
		return (p_num + p_den - 1) / p_den;
	}
	static _ALWAYS_INLINE_ uint32_t division_round_up(uint32_t p_num, uint32_t p_den) {
		return (p_num + p_den - 1) / p_den;
	}
	static _ALWAYS_INLINE_ int64_t division_round_up(int32_t p_num, int64_t p_den) {
		return (p_num + p_den - 1) / p_den;
	}
	static _ALWAYS_INLINE_ uint64_t division_round_up(uint32_t p_num, uint64_t p_den) {
		return (p_num + p_den - 1) / p_den;
```
Arguments are prefixed with `p_`